bin/update-yarn

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# inspired by https://gist.github.com/maxrimue/16e272c4e0b8a8ac9dd5570e180e08c0
```

Comment:
nit: perhaps `update-yarn-lock-file` 

this could be obvious to everyone else, but i thought you were updating yarn at first :)